iam.tf

```diff
@@ -1,5 +1,12 @@
+locals {
+  rbac_create            = var.rbac_create != null ? var.rbac_create : true
+  service_account_create = var.service_account_create != null ? var.service_account_create : true
+  irsa_role_create       = var.irsa_role_create != null ? var.irsa_role_create : true
+  irsa_policy_enabled    = var.enabled && local.rbac_create && local.service_account_create && local.irsa_role_create == false
```

Comment:
I would like to avoid this heavy lifting that is already happening within the addon-irsa module. I think we are find just with `irsa_policy_enabled = var.irsa_policy_enabled != null ? var.irsa_policy_enabled : var.irsa_assume_role_enabled != true`. What do you think?